ballot/models.py

```diff
@@ -733,7 +754,9 @@ def refresh_all_ballot_item_measure_entries(self, contest_measure):
             }
         return results
 
-    def refresh_all_ballot_item_office_entries(self, contest_office):
+    @staticmethod
+    def refresh_all_ballot_item_office_entries(
```

Comment:
We only need to break variables onto a second line when there are enough incoming variables to cause a line wrap.